README.md

```diff
@@ -398,6 +399,9 @@ The following query parameters are supported
 - `receiveName`, only applies when `type` is `update` and is mandatory in this case. Specifies the smart contract receive name of the smart contract receive function being called.
 - `parameter`, only applies when `type` is `update` and is mandatory in this case. Specifies the smart contract parameter passed to the receive function. Given as a hex string.
 - `executionNRGBuffer`, optionally and only applies when `type` is `update`. Specifies a buffer in percentage of how much of the energy for invoking the contract should be added to the returned cost. Given as an   integer. If not provided, a default of 20% is used.
+- `tokenId`, the tokenId of the plt token in the plt token update transaction. The query parameter `tokenId` is mandatory when `type` is `tokenUpdate` and otherwise ignored.
+- `listOperationsSize`, the size of the cbor encoded list of plt operations in the plt token update transaction. The query parameter `listOperationsSize` is mandatory when `type` is `tokenUpdate` and otherwise ignored.
+- `tokenOperationTypeCount`, is a URL encoded object counting the occurrence of each operation type in the plt token update transaction. Currently, the encoded object can include any of the following keys `transfer`, `mint`, `burn`, `listOperation`, and/or `pauseOrUnpause`. As each token operation type has a specific cost associated to it, it is necessary to know how often each type occurs in the cbor encoded list of plt operations. The query parameter `tokenOperationTypeCount` is mandatory when `type` is `tokenUpdate` and otherwise ignored.
```

Comment:
```suggestion
- `tokenOperationTypeCount`, is a URL-encoded JSON object counting the occurrence of each operation type in the PLT update transaction. Currently, the encoded object can include any of the following keys `transfer`, `mint`, `burn`, `listOperation`, and/or `pauseOrUnpause`. As each token operation type has a specific cost associated to it, it is necessary to know how often each type occurs in the CBOR encoded list of PLT operations. The query parameter `tokenOperationTypeCount` is mandatory when `type` is `tokenUpdate` and otherwise ignored.
```